src/components/Collection/CollectionList/DeleteCollectionAction.tsx

```diff
@@ -0,0 +1,18 @@
+import useTranslation from 'next-translate/useTranslation';
+
+import PopoverMenu from 'src/components/dls/PopoverMenu/PopoverMenu';
+import { deleteCollection } from 'src/utils/auth/api';
+
+const DeleteCollectionAction = ({ collectionId, onDone }) => {
+  const { t } = useTranslation();
+
+  const onMenuClicked = () => {
+    deleteCollection(collectionId).then(() => {
+      onDone();
+    });
```

Comment:
What happens when there is an issue with the API? I think we should show a toast/error message